src/commands/ActivityTrackerCommand.ts

```diff
@@ -42,7 +42,7 @@ class ActivityTrackerCommand extends Command {
 				await statsAll(interaction);
 			}
 		} else if (group === "admin") {
-			if (await !hasAdminRole(interaction)) {
+			if (!hasAdminRole(interaction)) {
```

Comment:
??

Reply:
vscode told me it wasn't necessary. I think a promise in an if statement always gets awaited (?)

Reply:
It is necessary, the old code was wrong too, the `!` turned the promise into a boolean and that couldn't be awaited. This one is correct as it's first awaited and then inverted.

Reply:
oh okay yeah